confidant/utils/misc.py

```diff
@@ -54,3 +55,24 @@ def utcnow():
     """
     now = datetime.utcnow()
     return now.replace(tzinfo=pytz.utc)
+
+
+def prevent_xss(pre_xss_response):
```

Comment:
recommend to change this as a decorator, just an example like this

Reply:
```
def prevent_xss_decorator(func):
    @wraps(func)
    def wrapper(*args, **kwargs):
        # Call the original function to get the response
        pre_xss_response = func(*args, **kwargs)
        
        # Apply XSS prevention
        response = make_response(pre_xss_response)
        response.headers['Content-Type'] = 'application/json'
        response.headers['X-XSS-Protection'] = '1; mode=block'
        response.headers['X-Content-Type-Options'] = 'nosniff'
        response.headers['Referrer-Policy'] = 'no-referrer'
        
        return response
    return wrapper
```

Reply:
then use it in the routes that can be applied `@prevent_xss_decorator` above the route declaration